fast64_internal/f3d/f3d_material.py

```diff
@@ -2513,42 +2522,49 @@ class RDPSettings(bpy.types.PropertyGroup):
         items=enumAlphaDither,
         default="G_AD_NOISE",
         update=update_node_values_with_preset,
+        description="Applies your choice dithering type to output frambuffer alpha"
```

Comment:
Typo "frambuffer", also I might explain that dithering is how to convert higher precision values to the lower precision for the framebuffer.